config/tags.yml

```diff
@@ -17,7 +17,6 @@
 - personal statements
 - qualifications
 - references
-- returning to teaching
```

Comment:
we should probably add a redirect from `/blog/tag/returning-to-teaching` to the TV page just for completeness @philspray ?